configuration/facebook_pipeline_config.json

```diff
@@ -33,19 +39,26 @@
         {"Name": "facebook_s08e01_BarnaamijkaHage", "PostIDs": [
           "325301151525528_697701827618790",
           "325301151525528_699376604117979",
-          "325301151525528_700208297368143"
+          "325301151525528_700208297368143",
+          "325301151525528_700986553956984"
+        ]},
+        {"Name": "facebook_s08e02_BarnaamijkaHage", "PostIDs": [
+          "325301151525528_703164037072569"
         ]}
       ]
     },
     {
       "SourceType": "Facebook",
-      "PageID": "M2A-Media-Communication-Consultancy-548414038603623",
+      "PageID": "548414038603623",
```

Comment:
(I'm changing this to the short id here because Facebook rejects the full name of the page as a page id)